packages/dds/tree/src/test/feature-libraries/sequence-field/markListFactory.spec.ts

```diff
@@ -129,12 +131,14 @@ describe("SequenceField - MarkListFactory", () => {
 			type: "Revive",
 			detachedBy,
 			detachIndex: 0,
+			content: fakeRepairData,
```

Comment:
Instead of using an empty array everywhere we want to use an array with unique data so we can check that the concatenation happens correctly.